plugins/woocommerce/tests/e2e-pw/tests/merchant/order-bulk-edit.spec.js

```diff
@@ -66,24 +66,24 @@ test.describe( 'Bulk edit orders', () => {
 
 	test( 'can bulk update order status', async ( { page } ) => {
 		await page.goto( 'wp-admin/admin.php?page=wc-orders' );
-
+		
 		// expect order status 'processing' to show
-		await expect( page.locator( `#order-${ orderId1 }`).getByText( 'Processing') ).toBeVisible();
-		await expect( page.locator( `#order-${ orderId2 }`).getByText( 'Processing') ).toBeVisible();
-		await expect( page.locator( `#order-${ orderId3 }`).getByText( 'Processing') ).toBeVisible();
-		await expect( page.locator( `#order-${ orderId4 }`).getByText( 'Processing') ).toBeVisible();
-		await expect( page.locator( `#order-${ orderId5 }`).getByText( 'Processing') ).toBeVisible();
+		await expect( page.locator( `:is(#order-${ orderId1 }, #post-${ orderId1 })` ).getByText( 'Processing' ) ).toBeVisible();
+		await expect( page.locator( `:is(#order-${ orderId2 }, #post-${ orderId2 })` ).getByText( 'Processing' ) ).toBeVisible();
+		await expect( page.locator( `:is(#order-${ orderId3 }, #post-${ orderId3 })` ).getByText( 'Processing' ) ).toBeVisible();
+		await expect( page.locator( `:is(#order-${ orderId4 }, #post-${ orderId4 })` ).getByText( 'Processing' ) ).toBeVisible();
+		await expect( page.locator( `:is(#order-${ orderId5 }, #post-${ orderId5 })` ).getByText( 'Processing' ) ).toBeVisible();
 
 		await page.locator( '#cb-select-all-1' ).click();
 		await page.locator( '#bulk-action-selector-top' ).selectOption( 'Change status to completed' );
 		await page.locator('#doaction').click();
 
 		// expect order status 'completed' to show
-		await expect( page.locator( `#order-${ orderId1 }`).getByText( 'Completed') ).toBeVisible();
-		await expect( page.locator( `#order-${ orderId2 }`).getByText( 'Completed') ).toBeVisible();
-		await expect( page.locator( `#order-${ orderId3 }`).getByText( 'Completed') ).toBeVisible();
-		await expect( page.locator( `#order-${ orderId4 }`).getByText( 'Completed') ).toBeVisible();
-		await expect( page.locator( `#order-${ orderId5 }`).getByText( 'Completed') ).toBeVisible();
+		await expect( page.locator( `:is(#order-${ orderId1 }, #post-${ orderId1 })`).getByText( 'Completed' ) ).toBeVisible();
+		await expect( page.locator( `:is(#order-${ orderId2 }, #post-${ orderId2 })`).getByText( 'Completed' ) ).toBeVisible();
+		await expect( page.locator( `:is(#order-${ orderId3 }, #post-${ orderId3 })`).getByText( 'Completed' ) ).toBeVisible();
+		await expect( page.locator( `:is(#order-${ orderId4 }, #post-${ orderId4 })`).getByText( 'Completed' ) ).toBeVisible();
+		await expect( page.locator( `:is(#order-${ orderId5 }, #post-${ orderId5 })`).getByText( 'Completed' ) ).toBeVisible();
```

Comment:
This is a repetitive assertion that depends on certain data such as orderId1, orderId2, etc. I think we can keep it simpler and avoid duplication by doing:
```
const orderIds = [orderId1, orderId2, orderId3, orderId4, orderId5];

for (const orderId of orderIds) {
    await expect(
        page.locator(`:is(#order-${orderId}, #post-${orderId})`).getByText('Completed')
    ).toBeVisible();
}
```